From jeffm@suse.com  Thu Oct  8 14:45:31 2009
From: Jeff Mahoney <jeffm@suse.com>
Date: Thu, 08 Oct 2009 17:32:42 -0400
Subject: Staging: iio: Don't build on s390
To: Greg KH <gregkh@suse.de>
Message-ID: <4ACE5A7A.4010907@suse.com>


The IIO core expects request_irq to work, which doesn't appear to exist
on s390.

Signed-off-by: Jeff Mahoney <jeffm@suse.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/iio/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/iio/Kconfig
+++ b/drivers/staging/iio/Kconfig
@@ -4,6 +4,7 @@
 
 menuconfig IIO
 	tristate "Industrial I/O support"
+	depends on !S390
 	---help---
 	  The industrial I/O subsystem provides a unified framework for
 	  drivers for many different types of embedded sensors using a
